microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -5,7 +5,7 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
```

Comment:
line 2: change from:
title: View page resources
to:
title: View the resource files that make up a webpage

Reply:
done in local